spec/views/badges/index.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe 'badges/index.html.erb', type: :view do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.